cristin-organization/src/test/java/no/unit/nva/cristin/organization/organization.feature

```diff
@@ -0,0 +1,61 @@
+Feature: API tests for Cristin Organization retrieve and search
+
+  Background:
+    * def SERVER_URL = 'https://api.dev.nva.aws.unit.no'
+    * def testOrganizationNameSearchTerm = 'univers'
+    * def CRISTIN_BASE = SERVER_URL + '/karate-cristin'
+    * def illegalIdentifier = 'illegalIdentifier'
+    * def nonExistingOrganizationId = '0.1.2.3'
+    Given url CRISTIN_BASE
+
+  Scenario: GET organization returns list of search results
+    Given  path '/organization'
+    And param query = illegalIdentifier
+    When method GET
+    Then status 200
+    And match response.hits == '#array'
+    And match response.size == 0
+
+  Scenario: GET returns 400 status Bad request when path parameter identifier is missing
+    Given path '/organization/+'
+    When method GET
+    Then status 400
+    And match response.title == 'Bad Request'
+    And match response.status == 400
+    And match response.detail == 'Invalid path parameter for identifier, needs to be a number'
+
+  Scenario: GET returns 400 status Bad request when requesting illegal organization identifier
+    Given path '/organization/' + illegalIdentifier
+    When method GET
+    Then status 400
+    And match response.title == 'Bad Request'
+    And match response.status == 400
+    And match response.detail == 'Invalid path parameter for identifier, needs to be a number'
+
+  Scenario: GET returns 400 status Bad request when requesting without any query parameter
+    Given path '/organization/'
+    When method GET
+    Then status 400
+    And match response.title == 'Bad Request'
+    And match response.status == 400
+    And match response.detail == 'Parameter \'query\' is missing or invalid. May only contain alphanumeric characters, dash, comma, period and whitespace'
```

Comment:
We should discuss one day if we should call every query using the same param 'query' and how we validate query across endpoints. One name for all or each its own, i dont know. Is a project search a query? Is a person or organization search also a query? If it is, its easier to validate what is sent from client, or we have to override query param validation if we use abstract classes for handling shared handler logic